### DISCUSSION TITLE
How to build WiX Custom Action projects without installing the toolset

### DISCUSSION THREAD

**Discussion by Rakesh1234567** [wixtoolset/issues#6481] [Questions]
Hi,
We want to build Custom action instead of installing toolset just refer toolset from directory. How we can achieve that?

Regards,
Rakesh

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions

**Comment by BMurri** [comment]
You need to, at minimum, define the property `WixTasksPath` (to the full path of the `WixTasks.dll` file) and either the `WixSdkPath` or the `WixToolPath` full-directory-path properties (preferably WixSdkPath, defining only WixToolPath will only help if that directory is a sibling of the `sdk` directory), before including `$(WixCATargetsPath)` (which happens to need to be the full path of the `wix.ca.targets` file).
